spec/support/capybara.rb

```diff
@@ -1,8 +1,28 @@
 require "capybara/rails"
 require "capybara/rspec"
+require "selenium-webdriver"
+
+Capybara.register_driver(:remote_firefox) do |app|
+  url = "http://selenium:4444/wd/hub"
+  options = Selenium::WebDriver::Firefox::Options.new
+  options.add_argument("--headless")
+  Capybara::Selenium::Driver.new(
+    app,
+    browser: :remote,
+    options:,
+    url:
+  )
+end
 
 Capybara.default_driver = :rack_test
-Capybara.javascript_driver = :selenium_headless
+
+if ENV.fetch("IN_DOCKER", nil)
```

Comment:
fetchでデフォルト値なんていれる必要ないよね
```suggestion
if ENV["IN_DOCKER"]
```

Reply:
INなのか、よくわからない。
USING_DOCKERとかなのかもしれない。

Reply:
Fixed at d48f24dc597cc79bb6c6e98362f296b12c0346eb
IN_DOCKERという名前の環境変数を使うのはやめました